src/site/content/en/lighthouse-best-practices/errors-in-console/index.md

```diff
@@ -2,82 +2,75 @@
 layout: post
 title: Browser errors were logged to the console
 description: |
-  Learn about `errors-in-console` audit.
+  Learn how to identify and fix browser errors.
 web_lighthouse:
   - errors-in-console
+updated: 2019-08-28
 ---
 
-Lighthouse flags any browser errors logged to the console:
+Most browsers ship with built-in developer tools.
+These developer tools usually include a [console](https://developers.google.com/web/tools/chrome-devtools/console/).
+The console gives you information about the page that's currently running.
+
+Messages logged in the console have a severity level:
+either `Verbose`, `Info`, `Warning`, or `Error`.
+An `Error` message means there's a problem on your page that you need to resolve.
+
+## How the Lighthouse browser error audit fails
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/) flags all browser errors logged to the console:
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="errors-in-console.png" alt="Lighthouse audit showing browser errors in the console">
-  <figcaption class="w-figcaption">
-    Figure 1. Browser errors logged to the console.
-  </figcaption>
+  <img class="w-screenshot" src="errors-in-console.png" alt="Lighthouse audit showing browser errors in the console">
 </figure>
 
-## How this audit fails
+{% include 'content/lighthouse-best-practices/scoring.njk' %}
 
-Most browsers ship with built-in developer tools.
-These developer tools usually include a console.
-The console gives you information about a page while that page is running.
-Ultimately,
-the messages you see in the console either come from the web developers who built the page,
-or the browser itself.
```

Comment:
If we're going to conceptually explain the Console, I think this sentence is important. It helps people understand where the messages are coming from. Either from the web developers who built the page, by means of `console.error()`, or from the browser itself.

But considering that we're starting to gravitate towards a more intermediate audience, I'm not too worried about conceptually explaining the Console. The Console is one of the first things that web developers learn about.

Reply:
Yeah, I really went back and forth on this sentence. I'm OK with reinstating it.